addons/crm/tests/test_crm_pls.py

```diff
@@ -210,7 +210,7 @@ def test_predictive_lead_scoring(self):
         if not loaded_demo_data(self.env):
             expected_proba = 35.19
         else:
-            expected_proba = 36.65
+            expected_proba = 35.58
```

Comment:
Meh

Reply:
Had to change it because of the changes in the demo data 
so I took advice from JBN and changed the testcase